### PR TITLE
update for reader

### PR DESCRIPTION
**Pull request by liamcindy**
1.update for sprite3d cullface
2.sync js code

### CODE REVIEW

**Comment by liamcindy** [comment]
@pandamicro  Please review this PR, thanks.

**Comment by pandamicro** [comment]
[ci rebuild]

File: cocos/scripting/js-bindings/script/jsb_opengl_constants.js [hunk 1]
```diff
@@ -826,6 +826,11 @@ gl.WRITEONLY_RENDERING_QCOM	= 0x8823;
 gl.WRITE_ONLY_OES	= 0x88b9;
 gl.Z400_BINARY_AMD	= 0x8740;
 gl.ZERO	= 0x0;
+gl.FRONT = 0x0404;
+gl.BACK = 0x0405;
```
Comment (pandamicro) [inline cocos/scripting/js-bindings/script/jsb_opengl_constants.js]: gl.FRONT, gl.BACK and gl.FRONT_AND_BACK already exist, please remove them.

And the list is ordered by alphabetic order, so please respect the order

Comment (liamcindy) [inline cocos/scripting/js-bindings/script/jsb_opengl_constants.js]: OK

**Pull request merged**